jenkins.sh

```diff
@@ -6,5 +6,6 @@ git clean -fdx
 
 bundle install --path "${HOME}/bundles/${JOB_NAME}" --deployment
 
+bundle exec govuk-lint-ruby --diff --cached
```

Comment:
Follow this format: https://github.com/alphagov/signonotron2/blob/34705e34e16b8d8fb895ae325ccdcc5338fa048f/jenkins.sh

If you've done the work to remove all violations, you don't need to run the tool in diff mode in CI as everything should be correct.